cargo-shuttle/src/lib.rs

```diff
@@ -152,12 +152,21 @@ impl Shuttle {
 
         // 3. Confirm the project directory
         let path = if interactive {
-            println!("Where should we create this project?");
-            let directory_str: String = Input::with_theme(&theme)
-                .with_prompt("Directory")
-                .default(".".to_owned())
-                .interact()?;
-            println!();
+            let directory_str: String;
+            if project_args.working_directory.as_os_str().is_empty() {
+                println!("Where should we create this project?");
+                directory_str = Input::with_theme(&theme)
+                    .with_prompt("Directory")
+                    .default(".".to_owned())
+                    .interact()?;
+                println!();
+            } else {
+                directory_str = project_args
+                    .working_directory
+                    .into_os_string()
+                    .into_string()
+                    .unwrap();
+            }
```

Comment:
So I tested this now, and it seems that if you pass in the path now it will always create the project in the current directory. However the way it was, if you passed the path in the interactive prompt it would actually create the project in that directory if it exists. 

Would it be better to simply pass in the `args.path`? At least then it will show up in the prompt, and not be ignored. Though I think we should fix both problems before we merge this, but please take your time. We can also look into it if you don't think you will be able to for a while.
```suggestion
            let path = args.path.to_str().context("path arg should always be set")?;

            println!("Where should we create this project?");
            let directory_str: String = Input::with_theme(&theme)
                .with_prompt("Directory")
                .default(path.to_owned())
                .interact()?;
            println!();
```

Reply:
In addition to the above suggestion, I think all we would need to do to fix the second issue (the directory not being created if path is passed in to init command) is to change this: https://github.com/shuttle-hq/shuttle/blob/9f7a48287ea452c578e98d72d4267c62a5208807/cargo-shuttle/src/args.rs#L206 to use the `parse_init_path` function rather than the `parse_path` function.